src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -479,6 +479,26 @@ private IOperation CreateBoundUnconvertedAddressOfOperatorOperation(BoundUnconve
                 boundUnconvertedAddressOf.WasCompilerGenerated);
         }
 
+        private IOperation CreateBoundAttributeOperation(BoundAttribute boundAttribute)
+        {
+            var isImplicit = boundAttribute.WasCompilerGenerated;
```

Comment:
consider naming e.g. `isAttributeImplicit` or similar to help indicate that passing `isImplicit: true` in several cases below is intended.